domain_specific.tex

```diff
@@ -0,0 +1,78 @@
+SymPy includes several packages that allow users to solve domain specific
+problems. For example, a comprehensive physics package is included that is
+useful for solving problems in classical mechanics, optics, and quantum
+mechanics along with support for manipuating physical quantities with units.
+
+\subsection{Vector Algebra}
+
+The vector package provides reference frame, time, and space aware vector and
+dyadic objects that allow for three dimensional operations such as addition,
+subtraction, scalar multiplication, inner and outer products, cross products,
+etc. Both of these objects can be written in very compact notation that make it
+easy to express the vectors and dyadics in terms of multiple reference frames
+with arbitrarily defined relative orientations. The vectors are used to specify
+the positions, velocities, and accelerations of points, orientations, angular
+velocities, and angular accelerations of reference frames, and force and
+torques. The dyadics are essentially reference frame aware 3 x 3 tensors. The
+vector and dyadic objects can be used for any one, two, or three dimensional
+vector algebra and provide a strong framework to build physics and engineering
+tools atop.
+
+\begin{listing}
+  \begin{minted}{pycon}
+>>> A = ReferenceFrame('A')
+>>> B = ReferenceFrame('B')
+>>> C = ReferenceFrame('C')
+>>> B.orient(A, 'body', (1, 2, 3), 'xzx')
```

Comment:
You should add a comment or description to the caption of what the orient lines do. It's not immediately obvious to me what this means.